aybapi/impl/client.py

```diff
@@ -0,0 +1,47 @@
+import typing
+
+# Importing implemented models
+from .bot import Bot
+from .category import Category
+
+# Importing helper methods and utils
+from ..utils.endpoints import *
+from ..utils.fetch import fetch
+
+# Importing base class of Client model
+from ..base.client import Client as ClientBase
+
+
+class Client(ClientBase):
+    def fetch_stats(self) -> typing.Optional[dict]:
+        """Fetches the overall site status available to the public
+
+        Returns
+        -------
+        API response: typing.Dict
+                The API response containing the site status information.
+        """
+        data = fetch(BASE_URL + STATS)
+        return data
+
+    def fetch_bot(self, id) -> Bot:
+        """Fetches a specific bot according to it's ID
+
+        Returns
+        -------
+        bot: aybapi.Bot
+                AYBAPI bot object containing bot information from API response.
+        """
+        data = fetch(BASE_URL + BOT.format(id))
```

Comment:
why aren't you using `requests`' ability to format query strings properly?

```py
with requests.get(url, params={"id": id})
```
That would then also handle any cases where the user could potentially alter the request being used by adding other characters into the input, which may or may not be a vulerability if you do not clearly document that this is how you do it.